src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -0,0 +1,82 @@
+package de.retest.web;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import de.retest.ui.DefaultValueFinder;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+
+public class DefaultValuesProvider implements DefaultValueFinder {
+
+	public static final String DEFAULTS_FILE_PATH = "/defaults.yaml";
+
+	private final Map<String, Map<String, String>> defaultValues;
+
+	public DefaultValuesProvider() {
+		try ( final InputStream url = getClass().getResourceAsStream( DEFAULTS_FILE_PATH ) ) {
+			defaultValues = readAttributesConfigFromFile( url );
+		} catch ( final Exception e ) {
+			// TODO Own exception
+			throw new AttributesConfigLoadException( DEFAULTS_FILE_PATH, e );
+		}
+	}
+
+	private Map<String, Map<String, String>> readAttributesConfigFromFile( final InputStream in ) throws IOException {
```

Comment:
This can be a static method.

Reply:
Could be, but would convey a wrong notion. This is a method that should only be called internally...